### DISCUSSION TITLE
스터디 일정 공지 : 23년 06월 07일 수요일 8시

### DISCUSSION THREAD

**Discussion by Emma-Hyejin** [wa-reureu-studyroom/toy-project-dailymusic#10] [Announcements]
* SSO  공부 
* Spotify API 공부 
* 깃 공부

**Comment by yeon0226** [comment]
study~~~~~~~📖🫠

**Comment by HyoJeong-Park** [comment]
오늘 서터디 요약

1. SSO, Spotify API 공부는 추가 시간이 필요함.
2. 플레이리스트 페이지, 마이페이지, 커뮤니티 페이지에서 할 수 있는 부분 나누어서 6월 11일(일)까지 
3. 깃에서 fetch로 받아서 merge 시키고 충돌 나는 거 수정해주고 push로 올리자.. 올렸으면 pr도 하자.. (main 브랜치 절대 돈땃쥐)

p.s 혜진쓰가 집사면 같이 살기로 했다. (무튼 그렇다!)
